packages/google_maps_flutter/google_maps_flutter_web/lib/src/markers.dart

```diff
@@ -147,4 +147,10 @@ class MarkersController extends GeometryController {
       markerId,
     ));
   }
+
+  void _hideAllMarkerInfoWindow() {
+    _markerIdToController.values
+        .where((controller) => controller?.infoWindowShown)
```

Comment:
If controller is `null`, then the function returns `null` instead of `false`, I think that's a problem.

Reply:
added a check